### DISCUSSION TITLE
How do I protect mod blocks on claim?

### DISCUSSION THREAD

**Discussion by oVictow** [GriefPrevention/GriefPrevention#1073] [Q&A] [answered]
I have a public server and I use GriefPrevention to protect claims.

My players can access any block interface in any mod, how can I fix it?
Sorry for my english, I am Brazilian...

**Comment by RoboMWM** [comment]
Check the config for the "mod" options. There aren't many, and they only exist in older versions (as I assume you're running an older server version.)

**Comment by oVictow** [comment]
I'm running version 1.12.2, I tried to use the options for mods, but they didn't work.

**Comment by RoboMWM** [comment]
Ok, there's still some old documentation for the mod section currently in the config. Since it doesn't exist anymore I'll move it out into its own page.

**Comment by RoboMWM** [comment, marked as answer]
Was in Setup and Configuration, now in [Legacy options](https://github.com/TechFortress/GriefPrevention/wiki/Legacy-options)
